src/main/java/com/conveyal/datatools/manager/models/transform/FeedTransformTarget.java

```diff
@@ -0,0 +1,32 @@
+package com.conveyal.datatools.manager.models.transform;
+
+import java.io.File;
+
+/**
+ * This class collects the different targets that a {@link FeedTransformation} can apply to:
+ * - zip file and
+ * - snapshot ID (which is a proxy for the database namespace)
+ *
+ * Generally, only one of these fields should contain a value. This class exists so that we can keep
+ * {@link FeedTransformation#transform} parameters constant for polymorphic subclasses.
+ */
+public class FeedTransformTarget {
+    public File gtfsFile;
+    public String snapshotId;
+
+    public FeedTransformTarget(File gtfsFile) {
+        this.gtfsFile = gtfsFile;
+    }
+
+    public FeedTransformTarget(String targetSnapshotId) {
+        this.snapshotId = targetSnapshotId;
+    }
+
```

Comment:
This might need to be split into sub classes, such as `FeedTransformFileTarget` and `FeedTransformDbTarget`, to parallel the Zip and Db transforms.

Reply:
Also, db operations are not valid with zip targets and vice-versa.

Reply:
After reading implementations, I am going to request the above split.

Reply:
Great suggestion, @binh-dam-ibigroup.